test/ir/pir/cinn/symbolic/CMakeLists.txt

```diff
@@ -46,4 +46,17 @@ if(WITH_GPU)
   set_tests_properties(test_sub_graph_for_backend PROPERTIES LABELS
                                                              "RUN_TYPE=CINN")
 
+  add_test(
+    NAME test_sub_graph_for_frontend
+    COMMAND
+      ${CMAKE_COMMAND} -E env
+      PYTHONPATH=${CMAKE_BINARY_DIR}:${CMAKE_BINARY_DIR}/python/:$ENV{PYTHONPATH}
+      FLAGS_cinn_convert_dynamic_to_static_dim=S0:64
```

Comment:
FLAGS_cinn_convert_dynamic_to_static_dim -> FLAGS_cinn_convert_dynamic_dim_to_static

Reply:
Considering others can use D2S right now, maybe we can use `FLAGS_cinn_convert_dynamic_to_static_dim` temporarily.